stable/rocketchat/README.md

```diff
@@ -13,12 +13,13 @@ $ helm install stable/rocketchat --set mongodb.mongodbPassword=$(echo -n $(opens
 ## Introduction
 
 This chart bootstraps a [Rocket.Chat](https://rocket.chat/) Deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. It provisions a fully featured Rocket.Chat installation.
-For more information on Rocket.Chat and its capabilities, see its [documentation](https://rocket.chat/docs/).
+
+In addition, this chart supports scaling of Rocket.Chat for increased server capacity and high availability.  For more information on Rocket.Chat and its capabilities, see its [documentation](https://rocket.chat/docs/).
 
 ## Prerequisites Details
 
 The chart has an optional dependency on the [MongoDB](https://github.com/kubernetes/charts/tree/master/stable/mongodb) chart.
-By default, the MongoDB chart requires PV support on underlying infrastructure (may be disabled).
+By default, the MongoDB chart requires StatefulSet support on underlying infrastructure (may be overridden).
```

Comment:
Might be worth keeping PV.. as some k8s setups might not have configured a provider for provisioning volumes

Reply:
Reverted to retain PV.  Thanks.